setup.py

```diff
@@ -24,7 +24,7 @@
   "rich==13.7.1",
   "tenacity==9.0.0",
   "tqdm==4.66.4",
-  "transformers==4.46.3",
```

Comment:
why did you downgrade transformers? is this intentional?

Reply:
it was intentional, I was getting errors with the latest version of transformers during compilation, I have found a workaround and no longer uses the 4.43